app/models/user.rb

```diff
@@ -54,7 +54,7 @@ def get_team_project_count
   end
 
   def get_team_count
-    if staff?      
+    if staff?
       teams.count
```

Comment:
this should be handled in ability. teams.count should work for all users if we have defined it in ability.rb